.prettierrc

```diff
@@ -6,7 +6,7 @@
   "quoteProps": "as-needed",
   "trailingComma": "none",
   "bracketSpacing": true,
-  "arrowParens": "always",
+  "arrowParens": "avoid",
```

Comment:
Parentheses are required in arrow functions when there are multiple parameters or when the type of the parameter is explicitly set. Otherwise, they are redundant.

Would you rather have:
```TypeScript
this.sectionForms.some(form => form.invalid)
```
or:
```TypeScript
this.sectionForms.some((form) => {
  return form.invalid;
})
```
?